tasks/benchmark.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+namespace :wings do
+  unless Module.const_defined?(:APP_RAKEFILE)
+    APP_RAKEFILE = File.expand_path("Rakefile", Pathname.new(__dir__).join("..", "..", "hyrax-webapp"))
+    load 'rails/tasks/engine.rake'
+  end
+  task benchmark_save: :environment do
+    # Base Case: 1.4 / Second
+    Rails.logger = Logger.new(STDOUT)
```

Comment:
is this needed? it doesn't seem to be doing anything in my environment. even without it there's a TON of cruft being logged:

```
D, [2020-11-18T18:21:23.573140 #466] DEBUG -- :   Load LDP (3.7ms) http://fcrepo:8080/fcrepo/rest/dev/35/93/1c/cc/35931ccc-c3bb-47f7-b2ab-8f7108211836 Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:23.584696 #466] DEBUG -- :   Load LDP (4.0ms) http://fcrepo:8080/fcrepo/rest/dev/35/93/1c/cc/35931ccc-c3bb-47f7-b2ab-8f7108211836 Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:23.731514 #466] DEBUG -- :   Load LDP (3.7ms) http://fcrepo:8080/fcrepo/rest/dev/1f/44/3e/4b/1f443e4b-5611-44b5-9a2b-20d1bfa9145a Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:23.750155 #466] DEBUG -- :    (0.3ms)  BEGIN
D, [2020-11-18T18:21:23.750361 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/services/hyrax/noid.rb:9
D, [2020-11-18T18:21:23.752649 #466] DEBUG -- :   MinterState Load (1.2ms)  SELECT  "minter_states".* FROM "minter_states" WHERE "minter_states"."namespace" = $1 AND "minter_states"."template" = $2 LIMIT $3 FOR UPDATE  [["namespace", "default"], ["template", ".reeddeeddk"], ["LIMIT", 1]]
D, [2020-11-18T18:21:23.752846 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/services/hyrax/noid.rb:9
D, [2020-11-18T18:21:23.755246 #466] DEBUG -- :   MinterState Exists (0.4ms)  SELECT  1 AS one FROM "minter_states" WHERE "minter_states"."namespace" = $1 AND "minter_states"."id" != $2 LIMIT $3  [["namespace", "default"], ["id", 1], ["LIMIT", 1]]
D, [2020-11-18T18:21:23.755415 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/services/hyrax/noid.rb:9
D, [2020-11-18T18:21:23.757516 #466] DEBUG -- :   MinterState Update (1.1ms)  UPDATE "minter_states" SET "counters" = $1, "seq" = $2, "rand" = $3, "updated_at" = $4 WHERE "minter_states"."id" = $5  [["counters", "[{\"value\":0,\"max\":24139284},{\"value\":24139284,\"max\":48278568},{\"value\":48278568,\"max\":72417852},{\"value\":72417853,\"max\":96557136},{\"value\":96557136,\"max\":120696420},{\"value\":120696421,\"max\":144835704},{\"value\":144835704,\"max\":168974988},{\"value\":168974988,\"max\":193114272},{\"value\":193114272,\"max\":217253556},{\"value\":217253556,\"max\":241392840},{\"value\":241392840,\"max\":265532124},{\"value\":265532124,\"max\":289671408},{\"value\":289671408,\"max\":313810692},{\"value\":313810692,\"max\":337949976},{\"value\":337949976,\"max\":362089260},{\"value\":362089260,\"max\":386228544},{\"value\":386228544,\"max\":410367828},{\"value\":410367828,\"max\":434507112},{\"value\":434507112,\"max\":458646396},{\"value\":458646396,\"max\":482785680},{\"value\":482785681,\"max\":506924964},{\"value\":506924964,\"max\":531064248},{\"value\":531064248,\"max\":555203532},{\"value\":555203533,\"max\":579342816},{\"value\":579342817,\"max\":603482100},{\"value\":603482100,\"max\":627621384},{\"value\":627621384,\"max\":651760668},{\"value\":651760668,\"max\":675899952},{\"value\":675899952,\"max\":700039236},{\"value\":700039236,\"max\":724178520},{\"value\":724178520,\"max\":748317804},{\"value\":748317804,\"max\":772457088},{\"value\":772457088,\"max\":796596372},{\"value\":796596372,\"max\":820735656},{\"value\":820735657,\"max\":844874940},{\"value\":844874940,\"max\":869014224},{\"value\":869014224,\"max\":893153508},{\"value\":893153508,\"max\":917292792},{\"value\":917292793,\"max\":941432076},{\"value\":941432077,\"max\":965571360},{\"value\":965571360,\"max\":989710644},{\"value\":989710644,\"max\":1013849928},{\"value\":1013849928,\"max\":1037989212},{\"value\":1037989212,\"max\":1062128496},{\"value\":1062128496,\"max\":1086267780},{\"value\":1086267780,\"max\":1110407064},{\"value\":1110407064,\"max\":1134546348},{\"value\":1134546348,\"max\":1158685632},{\"value\":1158685632,\"max\":1182824916},{\"value\":1182824916,\"max\":1206964200},{\"value\":1206964201,\"max\":1231103484},{\"value\":1231103485,\"max\":1255242768},{\"value\":1255242768,\"max\":1279382052},{\"value\":1279382052,\"max\":1303521336},{\"value\":1303521336,\"max\":1327660620},{\"value\":1327660620,\"max\":1351799904},{\"value\":1351799904,\"max\":1375939188},{\"value\":1375939188,\"max\":1400078472},{\"value\":1400078473,\"max\":1424217756},{\"value\":1424217756,\"max\":1448357040},{\"value\":1448357040,\"max\":1472496324},{\"value\":1472496325,\"max\":1496635608},{\"value\":1496635608,\"max\":1520774892},{\"value\":1520774892,\"max\":1544914176},{\"value\":1544914176,\"max\":1569053460},{\"value\":1569053460,\"max\":1593192744},{\"value\":1593192746,\"max\":1617332028},{\"value\":1617332029,\"max\":1641471312},{\"value\":1641471312,\"max\":1665610596},{\"value\":1665610596,\"max\":1689749880},{\"value\":1689749880,\"max\":1713889164},{\"value\":1713889165,\"max\":1738028448},{\"value\":1738028448,\"max\":1762167732},{\"value\":1762167732,\"max\":1786307016},{\"value\":1786307016,\"max\":1810446300},{\"value\":1810446301,\"max\":1834585584},{\"value\":1834585585,\"max\":1858724868},{\"value\":1858724868,\"max\":1882864152},{\"value\":1882864152,\"max\":1907003436},{\"value\":1907003436,\"max\":1931142720},{\"value\":1931142721,\"max\":1955282004},{\"value\":1955282004,\"max\":1979421288},{\"value\":1979421288,\"max\":2003560572},{\"value\":2003560572,\"max\":2027699856},{\"value\":2027699857,\"max\":2051839140},{\"value\":2051839140,\"max\":2075978424},{\"value\":2075978424,\"max\":2100117708},{\"value\":2100117709,\"max\":2124256992},{\"value\":2124256993,\"max\":2148396276},{\"value\":2148396276,\"max\":2172535560},{\"value\":2172535560,\"max\":2196674844},{\"value\":2196674844,\"max\":2220814128},{\"value\":2220814128,\"max\":2244953412},{\"value\":2244953413,\"max\":2269092696},{\"value\":2269092696,\"max\":2293231980},{\"value\":2293231980,\"max\":2317371264},{\"value\":2317371264,\"max\":2341510548},{\"value\":2341510548,\"max\":2365649832},{\"value\":2365649832,\"max\":2389789116},{\"value\":2389789116,\"max\":2413928400},{\"value\":2413928400,\"max\":2438067684},{\"value\":2438067685,\"max\":2462206968},{\"value\":2462206968,\"max\":2486346252},{\"value\":2486346252,\"max\":2510485536},{\"value\":2510485536,\"max\":2534624820},{\"value\":2534624820,\"max\":2558764104},{\"value\":2558764104,\"max\":2582903388},{\"value\":2582903388,\"max\":2607042672},{\"value\":2607042672,\"max\":2631181956},{\"value\":2631181956,\"max\":2655321240},{\"value\":2655321240,\"max\":2679460524},{\"value\":2679460525,\"max\":2703599808},{\"value\":2703599808,\"max\":2727739092},{\"value\":2727739092,\"max\":2751878376},{\"value\":2751878376,\"max\":2776017660},{\"value\":2776017660,\"max\":2800156944},{\"value\":2800156944,\"max\":2824296228},{\"value\":2824296228,\"max\":2848435512},{\"value\":2848435512,\"max\":2872574796},{\"value\":2872574797,\"max\":2896714080},{\"value\":2896714080,\"max\":2920853364},{\"value\":2920853364,\"max\":2944992648},{\"value\":2944992648,\"max\":2969131932},{\"value\":2969131933,\"max\":2993271216},{\"value\":2993271217,\"max\":3017410500},{\"value\":3017410500,\"max\":3041549784},{\"value\":3041549784,\"max\":3065689068},{\"value\":3065689068,\"max\":3089828352},{\"value\":3089828352,\"max\":3113967636},{\"value\":3113967637,\"max\":3138106920},{\"value\":3138106920,\"max\":3162246204},{\"value\":3162246204,\"max\":3186385488},{\"value\":3186385488,\"max\":3210524772},{\"value\":3210524773,\"max\":3234664056},{\"value\":3234664056,\"max\":3258803340},{\"value\":3258803340,\"max\":3282942624},{\"value\":3282942624,\"max\":3307081908},{\"value\":3307081908,\"max\":3331221192},{\"value\":3331221192,\"max\":3355360476},{\"value\":3355360477,\"max\":3379499760},{\"value\":3379499760,\"max\":3403639044},{\"value\":3403639044,\"max\":3427778328},{\"value\":3427778328,\"max\":3451917612},{\"value\":3451917612,\"max\":3476056896},{\"value\":3476056897,\"max\":3500196180},{\"value\":3500196180,\"max\":3524335464},{\"value\":3524335465,\"max\":3548474748},{\"value\":3548474748,\"max\":3572614032},{\"value\":3572614033,\"max\":3596753316},{\"value\":3596753318,\"max\":3620892600},{\"value\":3620892600,\"max\":3645031884},{\"value\":3645031884,\"max\":3669171168},{\"value\":3669171168,\"max\":3693310452},{\"value\":3693310452,\"max\":3717449736},{\"value\":3717449736,\"max\":3741589020},{\"value\":3741589020,\"max\":3765728304},{\"value\":3765728304,\"max\":3789867588},{\"value\":3789867589,\"max\":3814006872},{\"value\":3814006872,\"max\":3838146156},{\"value\":3838146156,\"max\":3862285440},{\"value\":3862285441,\"max\":3886424724},{\"value\":3886424724,\"max\":3910564008},{\"value\":3910564008,\"max\":3934703292},{\"value\":3934703292,\"max\":3958842576},{\"value\":3958842576,\"max\":3982981860},{\"value\":3982981860,\"max\":4007121144},{\"value\":4007121145,\"max\":4031260428},{\"value\":4031260428,\"max\":4055399712},{\"value\":4055399712,\"max\":4079538996},{\"value\":4079538996,\"max\":4103678280},{\"value\":4103678280,\"max\":4127817564},{\"value\":4127817565,\"max\":4151956848},{\"value\":4151956849,\"max\":4176096132},{\"value\":4176096132,\"max\":4200235416},{\"value\":4200235416,\"max\":4224374700},{\"value\":4224374701,\"max\":4248513984},{\"value\":4248513984,\"max\":4272653268},{\"value\":4272653268,\"max\":4296792552},{\"value\":4296792552,\"max\":4320931836},{\"value\":4320931836,\"max\":4345071120},{\"value\":4345071120,\"max\":4369210404},{\"value\":4369210404,\"max\":4393349688},{\"value\":4393349689,\"max\":4417488972},{\"value\":4417488972,\"max\":4441628256},{\"value\":4441628256,\"max\":4465767540},{\"value\":4465767541,\"max\":4489906824},{\"value\":4489906824,\"max\":4514046108},{\"value\":4514046108,\"max\":4538185392},{\"value\":4538185392,\"max\":4562324676},{\"value\":4562324676,\"max\":4586463960},{\"value\":4586463960,\"max\":4610603244},{\"value\":4610603244,\"max\":4634742528},{\"value\":4634742528,\"max\":4658881812},{\"value\":4658881813,\"max\":4683021096},{\"value\":4683021097,\"max\":4707160380},{\"value\":4707160380,\"max\":4731299664},{\"value\":4731299664,\"max\":4755438948},{\"value\":4755438948,\"max\":4779578232},{\"value\":4779578232,\"max\":4803717516},{\"value\":4803717516,\"max\":4827856800},{\"value\":4827856800,\"max\":4851996084},{\"value\":4851996084,\"max\":4876135368},{\"value\":4876135369,\"max\":4900274652},{\"value\":4900274653,\"max\":4924413936},{\"value\":4924413936,\"max\":4948553220},{\"value\":4948553220,\"max\":4972692504},{\"value\":4972692504,\"max\":4996831788},{\"value\":4996831788,\"max\":5020971072},{\"value\":5020971072,\"max\":5045110356},{\"value\":5045110357,\"max\":5069249640},{\"value\":5069249640,\"max\":5093388924},{\"value\":5093388925,\"max\":5117528208},{\"value\":5117528209,\"max\":5141667492},{\"value\":5141667493,\"max\":5165806776},{\"value\":5165806776,\"max\":5189946060},{\"value\":5189946060,\"max\":5214085344},{\"value\":5214085345,\"max\":5238224628},{\"value\":5238224629,\"max\":5262363912},{\"value\":5262363912,\"max\":5286503196},{\"value\":5286503196,\"max\":5310642480},{\"value\":5310642480,\"max\":5334781764},{\"value\":5334781765,\"max\":5358921048},{\"value\":5358921048,\"max\":5383060332},{\"value\":5383060332,\"max\":5407199616},{\"value\":5407199616,\"max\":5431338900},{\"value\":5431338900,\"max\":5455478184},{\"value\":5455478184,\"max\":5479617468},{\"value\":5479617468,\"max\":5503756752},{\"value\":5503756752,\"max\":5527896036},{\"value\":5527896036,\"max\":5552035320},{\"value\":5552035320,\"max\":5576174604},{\"value\":5576174604,\"max\":5600313888},{\"value\":5600313888,\"max\":5624453172},{\"value\":5624453172,\"max\":5648592456},{\"value\":5648592456,\"max\":5672731740},{\"value\":5672731741,\"max\":5696871024},{\"value\":5696871024,\"max\":5721010308},{\"value\":5721010309,\"max\":5745149592},{\"value\":5745149592,\"max\":5769288876},{\"value\":5769288876,\"max\":5793428160},{\"value\":5793428160,\"max\":5817567444},{\"value\":5817567444,\"max\":5841706728},{\"value\":5841706729,\"max\":5865846012},{\"value\":5865846013,\"max\":5889985296},{\"value\":5889985297,\"max\":5914124580},{\"value\":5914124580,\"max\":5938263864},{\"value\":5938263864,\"max\":5962403148},{\"value\":5962403148,\"max\":5986542432},{\"value\":5986542432,\"max\":6010681716},{\"value\":6010681716,\"max\":6034821000},{\"value\":6034821000,\"max\":6058960284},{\"value\":6058960284,\"max\":6083099568},{\"value\":6083099568,\"max\":6107238852},{\"value\":6107238853,\"max\":6131378136},{\"value\":6131378137,\"max\":6155517420},{\"value\":6155517420,\"max\":6179656704},{\"value\":6179656704,\"max\":6203795988},{\"value\":6203795988,\"max\":6227935272},{\"value\":6227935272,\"max\":6252074556},{\"value\":6252074556,\"max\":6276213840},{\"value\":6276213841,\"max\":6300353124},{\"value\":6300353125,\"max\":6324492408},{\"value\":6324492408,\"max\":6348631692},{\"value\":6348631693,\"max\":6372770976},{\"value\":6372770976,\"max\":6396910260},{\"value\":6396910260,\"max\":6421049544},{\"value\":6421049544,\"max\":6445188828},{\"value\":6445188828,\"max\":6469328112},{\"value\":6469328112,\"max\":6493467396},{\"value\":6493467397,\"max\":6517606680},{\"value\":6517606680,\"max\":6541745964},{\"value\":6541745964,\"max\":6565885248},{\"value\":6565885248,\"max\":6590024532},{\"value\":6590024532,\"max\":6614163816},{\"value\":6614163817,\"max\":6638303100},{\"value\":6638303101,\"max\":6662442384},{\"value\":6662442384,\"max\":6686581668},{\"value\":6686581668,\"max\":6710720952},{\"value\":6710720952,\"max\":6734860236},{\"value\":6734860236,\"max\":6758999520},{\"value\":6758999520,\"max\":6783138804},{\"value\":6783138805,\"max\":6807278088},{\"value\":6807278088,\"max\":6831417372},{\"value\":6831417372,\"max\":6855556656},{\"value\":6855556656,\"max\":6879695940},{\"value\":6879695941,\"max\":6903835224},{\"value\":6903835224,\"max\":6927974508},{\"value\":6927974508,\"max\":6952113792},{\"value\":6952113792,\"max\":6976253076},{\"value\":6976253076,\"max\":7000392360},{\"value\":7000392360,\"max\":7024531644},{\"value\":7024531644,\"max\":7048670928},{\"value\":7048670928,\"max\":7072810000}]"], ["seq", 70], ["rand", "<2537 bytes of binary data>"], ["updated_at", "2020-11-18 18:21:23.755633"], ["id", 1]]
D, [2020-11-18T18:21:23.757756 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/services/hyrax/noid.rb:9
D, [2020-11-18T18:21:23.784965 #466] DEBUG -- :    (27.0ms)  COMMIT
D, [2020-11-18T18:21:23.785073 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/services/hyrax/noid.rb:9
D, [2020-11-18T18:21:23.809692 #466] DEBUG -- :   Load LDP (4.5ms) http://fcrepo:8080/fcrepo/rest/dev/x6/33/f1/00/x633f100h Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:23.821927 #466] DEBUG -- :   Sipity::Entity Load (0.3ms)  SELECT  "sipity_entities".* FROM "sipity_entities" WHERE "sipity_entities"."proxy_for_global_id" = $1 LIMIT $2  [["proxy_for_global_id", "gid://dassie/Monograph/x633f100h"], ["LIMIT", 1]]
D, [2020-11-18T18:21:23.822016 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/models/sipity.rb:22
D, [2020-11-18T18:21:23.822716 #466] DEBUG -- : [i18n-debug] en.hyrax.models.monograph => nil
D, [2020-11-18T18:21:23.822783 #466] DEBUG -- : [i18n-debug] en.hyrax.models.monograph => nil
D, [2020-11-18T18:21:23.918877 #466] DEBUG -- :   Load LDP (3.5ms) http://fcrepo:8080/fcrepo/rest/dev/1f/44/3e/4b/1f443e4b-5611-44b5-9a2b-20d1bfa9145a Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:23.932134 #466] DEBUG -- : Starting nested indexing of ID="x633f100h"
D, [2020-11-18T18:21:23.932201 #466] DEBUG -- : Starting indexing ID="x633f100h"
D, [2020-11-18T18:21:23.940184 #466] DEBUG -- :   Load LDP (5.2ms) http://fcrepo:8080/fcrepo/rest/dev/x6/33/f1/00/x633f100h Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:23.979718 #466] DEBUG -- :   Load LDP (4.2ms) http://fcrepo:8080/fcrepo/rest/dev/x6/33/f1/00/x633f100h Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:24.015941 #466] DEBUG -- :   Sipity::Entity Load (0.3ms)  SELECT  "sipity_entities".* FROM "sipity_entities" WHERE "sipity_entities"."proxy_for_global_id" = $1 LIMIT $2  [["proxy_for_global_id", "gid://dassie/Monograph/x633f100h"], ["LIMIT", 1]]
D, [2020-11-18T18:21:24.016063 #466] DEBUG -- :   ↳ /app/samvera/hyrax-engine/app/models/sipity.rb:22
D, [2020-11-18T18:21:24.026541 #466] DEBUG -- :   Load LDP (3.9ms) http://fcrepo:8080/fcrepo/rest/dev/1f/44/3e/4b/1f443e4b-5611-44b5-9a2b-20d1bfa9145a Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:24.037600 #466] DEBUG -- :   Load LDP (4.2ms) http://fcrepo:8080/fcrepo/rest/dev/1f/44/3e/4b/1f443e4b-5611-44b5-9a2b-20d1bfa9145a Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:24.044755 #466] DEBUG -- : [i18n-debug] en.hyrax.models.monograph => nil
D, [2020-11-18T18:21:24.044849 #466] DEBUG -- : [i18n-debug] en.hyrax.models.monograph => nil
D, [2020-11-18T18:21:24.051908 #466] DEBUG -- : Ending indexing ID="x633f100h"
D, [2020-11-18T18:21:24.054490 #466] DEBUG -- : Ending nested indexing of ID="x633f100h"
D, [2020-11-18T18:21:24.057727 #466] DEBUG -- :   Load LDP (3.1ms) http://fcrepo:8080/fcrepo/rest/dev/x6/33/f1/00/x633f100h/list_source Service: 436740
D, [2020-11-18T18:21:24.078103 #466] DEBUG -- :   Load LDP (2.5ms) http://fcrepo:8080/fcrepo/rest/dev/x6/33/f1/00/x633f100h/list_source Service: 436740
D, [2020-11-18T18:21:24.092412 #466] DEBUG -- :   Load LDP (3.6ms) http://fcrepo:8080/fcrepo/rest/dev/1f/44/3e/4b/1f443e4b-5611-44b5-9a2b-20d1bfa9145a Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
D, [2020-11-18T18:21:24.104037 #466] DEBUG -- :   Load LDP (4.0ms) http://fcrepo:8080/fcrepo/rest/dev/1f/44/3e/4b/1f443e4b-5611-44b5-9a2b-20d1bfa9145a Service: 436740
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
/usr/local/bundle/gems/rdf-3.0.13/lib/rdf/model/uri.rb:851: warning: URI.unescape is obsolete
      2.036  (± 0.0%) i/s -     11.000  in   5.418233s
```

Reply:
Yeah I can remove it. I actually tried to get that cruft to go away and couldn't figure out where it was coming from. 😓

Reply:
i can put in a follow up with better logger setup. Ruby 2.7 is also partially at fault here.